android/.project

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <projectDescription>
 	<name>celo</name>
```

Comment:
not sure how much this name matters, but should we have the name here and in `app/.project` consistent?